tests/integration_tests/adapters/binance/test_http_account.py

```diff
@@ -295,41 +331,16 @@ async def test_get_open_oco_orders_sends_expected_request(self, mocker):
         assert request["params"].startswith("recvWindow=5000&timestamp=")
 
     @pytest.mark.asyncio
-    async def test_account_sends_expected_request(self, mocker):
+    async def test_query_spot_account_info_sends_expected_request(self, mocker):
         # Arrange
         await self.client.connect()
         mock_send_request = mocker.patch(target="aiohttp.client.ClientSession.request")
 
         # Act
-        await self.api.account(recv_window=5000)
+        await self.api.query_spot_account_info(recv_window=5000)
 
         # Assert
         request = mock_send_request.call_args.kwargs
         assert request["method"] == "GET"
         assert request["url"] == "https://api.binance.com/api/v3/account"
         assert request["params"].startswith("recvWindow=5000&timestamp=")
-
-    @pytest.mark.asyncio
-    async def test_my_trades_sends_expected_request(self, mocker):
```

Comment:
Reason for removing this one?

Reply:
Looks like I just moved + renamed it, see [here](https://github.com/nautechsystems/nautilus_trader/pull/973/files#diff-8c426b2ebb52445175556df9700e0dbdf817fa2566237d6e1e62690406646344R197). I renamed the tests to match the consolidated endpoint names for consistency. This happens to be an endpoint where the name differs between Spot (`myTrades`) and Futures (`userTrades`) - I chose to use `userTrades` for the consolidated name.